{"uuid": "4a0ce849-8858-42e8-ae05-04a3de5a27e2", "counter": 46, "stdout": "\u001b[0;31mAn exception occurred during task execution. To see the full traceback, use -vvv. The error was: If you are using a module and expect the file to exist on the remote, see the remote_src option\u001b[0m\r\n\u001b[0;31mfatal: [demo03]: FAILED! => {\"changed\": false, \"msg\": \"Could not find or access 'host/httpd.conf.j2'\\nSearched in:\\n\\t/tmp/templates/host/httpd.conf.j2\\n\\t/tmp/host/httpd.conf.j2\\n\\t/tmp/templates/host/httpd.conf.j2\\n\\t/tmp/host/httpd.conf.j2 on the Ansible Controller.\\nIf you are using a module and expect the file to exist on the remote, see the remote_src option\"}\u001b[0m", "start_line": 52, "end_line": 54, "runner_ident": "b9a053ca-bbe7-4ab3-b04a-08ee46125fc2", "event": "runner_on_failed", "pid": 204378, "created": "2025-05-16T01:12:39.894273+00:00", "parent_uuid": "005056b7-f67d-7858-434d-00000000000f", "event_data": {"playbook": "/tmp/tmp3l2kwooa.yml", "playbook_uuid": "0dd191c0-577a-43aa-a085-26aea2338888", "play": "Configuraci\u00f3n de servidor web en Oracle Linux 9 (Host)", "play_uuid": "005056b7-f67d-7858-434d-000000000002", "play_pattern": "demo03", "task": "Copiar el archivo httpd.conf desde la plantilla espec\u00edfica de host", "task_uuid": "005056b7-f67d-7858-434d-00000000000f", "task_action": "template", "resolved_action": "ansible.builtin.template", "task_args": "", "task_path": "/tmp/tmp3l2kwooa.yml:75", "host": "demo03", "remote_addr": "demo03", "res": {"msg": "Could not find or access 'host/httpd.conf.j2'\nSearched in:\n\t/tmp/templates/host/httpd.conf.j2\n\t/tmp/host/httpd.conf.j2\n\t/tmp/templates/host/httpd.conf.j2\n\t/tmp/host/httpd.conf.j2 on the Ansible Controller.\nIf you are using a module and expect the file to exist on the remote, see the remote_src option", "exception": "Traceback (most recent call last):\n  File \"/usr/lib/python3.9/site-packages/ansible/plugins/action/template.py\", line 85, in run\n    source = self._find_needle('templates', source)\n  File \"/usr/lib/python3.9/site-packages/ansible/plugins/action/__init__.py\", line 1431, in _find_needle\n    return self._loader.path_dwim_relative_stack(path_stack, dirname, needle)\n  File \"/usr/lib/python3.9/site-packages/ansible/parsing/dataloader.py\", line 341, in path_dwim_relative_stack\n    raise AnsibleFileNotFound(file_name=source, paths=[to_native(p) for p in search])\nansible.errors.AnsibleFileNotFound: Could not find or access 'host/httpd.conf.j2'\nSearched in:\n\t/tmp/templates/host/httpd.conf.j2\n\t/tmp/host/httpd.conf.j2\n\t/tmp/templates/host/httpd.conf.j2\n\t/tmp/host/httpd.conf.j2 on the Ansible Controller.\nIf you are using a module and expect the file to exist on the remote, see the remote_src option\n", "_ansible_no_log": false, "changed": false}, "start": "2025-05-16T01:12:39.633421+00:00", "end": "2025-05-16T01:12:39.893652+00:00", "duration": 0.260231, "ignore_errors": null, "event_loop": null, "uuid": "4a0ce849-8858-42e8-ae05-04a3de5a27e2"}}